src/Validator/Rules/KnownTypeNames.php

```diff
@@ -22,30 +27,69 @@ class KnownTypeNames extends ValidationRule
 {
     public function getVisitor(ValidationContext $context): array
     {
-        $skip = static function (): VisitorOperation {
-            return Visitor::skipNode();
-        };
+        return $this->getVisitorInternal($context);
+    }
+
+    public function getSDLVisitor(SDLValidationContext $context): array
+    {
+        return $this->getVisitorInternal($context);
+    }
+
+    /**
+     * @param ValidationContext|SDLValidationContext $context
+     *
+     * @return Closure[]
```

Comment:
Right. Didn't exist back in October when I wrote the code.

Reply:
Got it 👍 The involved interfaces are going to change a bit in https://github.com/webonyx/graphql-php/pull/1059/files - I hope this will make the nature of the different contexts more clear in the future.